cdmsparkevents/eventloop.py

```diff
@@ -1,3 +1,44 @@
 """
 The main event loop for the event handler.
 """
+
+
+from kafka import KafkaConsumer
+import logging
+
+from cdmsparkevents.config import Config
+
+
+def get_kafka_consumer_from_config(config: Config) -> KafkaConsumer:
+    """
+    Get a Kafka consumer from a validated configuration. No further validation is performed.
+    
+    The consumer is set up with auto commit disabled.
+    """
+    return KafkaConsumer(
+        config.kafka_topic_jobs,
+        bootstrap_servers=config.kafka_bootstrap_servers.split(","),
+        group_id=config.kafka_group_id,
+        enable_auto_commit=False,
+        max_poll_interval_ms=config.kafka_max_poll_interval_ms,
+        # If no partion offsets exist for the group, go to the earliest record
+        auto_offset_reset="earliest",
```

Comment:
we don't need to define a `value_deserializer` function for `KafkaConsumer`? We will handle bytes messages later?

Reply:
yeah I'm just deserializing manually so it's easier to catch exceptions for deserialization and handle them separately

Reply:
It's just `json.loads(msg.value.decode())`

Reply:
👍